frontend/shared/components/src/members/components/platform-membership/SelectPlatformMembershipView.vue

```diff
@@ -213,34 +213,29 @@ function getTypeDateDescription(type: PlatformMembershipType) {
 }
 
 function getTypePriceDescription(type: PlatformMembershipType) {
-    const periodConfig = type.periods.get(platform.value.period.id);
-    if (!periodConfig) {
-        return 'Niet beschikbaar';
-    }
-
-    const priceConfig = periodConfig.getPriceForDate(new Date());
-
-    if (type.behaviour === PlatformMembershipTypeBehaviour.Days) {
-        return Formatter.price(priceConfig.pricePerDay) + ' per dag'
-    }
-
-    return Formatter.price(priceConfig.price);
+    return getPriceForDate(type, new Date());
 }
 
 function getTypePriceNormalPrice(type: PlatformMembershipType) {
+    return getPriceForDate(type, new Date(1));
+}
+
+function getPriceForDate(type: PlatformMembershipType, date: Date) {
     const periodConfig = type.periods.get(platform.value.period.id);
     if (!periodConfig) {
         return 'Niet beschikbaar';
     }
 
-    const priceConfig = periodConfig.getPriceForDate(new Date(1));
+    const priceConfig = periodConfig.getPriceConfigForDate(date);
 
     if (type.behaviour === PlatformMembershipTypeBehaviour.Days) {
         return Formatter.price(priceConfig.pricePerDay) + ' per dag'
     }
 
-    return Formatter.price(priceConfig.price);
-}
-
+    const tagIds = organization.value?.meta.tags ?? [];
```

Comment:
```suggestion
    const tagIds = selectedOrganization.value?.meta.tags ?? [];
```
-> je moet de prijs berekenen voor de geselecteerde vereniging, niet gewoon voor de vereniging van de session (als je via admin gaat, is er nooit een vereniging in de sessie, dus dan kan je een vereniging kiezen waarvoor je de platform membership wilt toevoegen)